src/pages/ProfilePage.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
+import PrivateNavbar from '../comps/PrivateNavbar.js';
+import Footer from '../comps/Footer.js';
+import SmallHeader from '../comps/SmallHeader.js';
 import '../css/App.css';
```

Comment:
I can give you dummy info for this based on some personas I've made

Reply:
Sure, but again, its a prototype of site layout. also the private pages are totally on the back burner - im prioritizing all public content first. When we start focussing on the private pages, then we will also have sign in flows that can be used to populate the profiles, too